pyega3/pyega3.py

```diff
@@ -110,62 +122,71 @@ def get_token(credentials):
         oauth_token = reply['access_token']
         logging.info("Authentication success for user '{}'".format(username))
     except Exception as ee:
-    	print(ee)
-    	sys.exit("Invalid username, password or secret key - please check and retry. If problem persists contact helpdesk on helpdesk@ega-archive.org")
+        print(ee)
+        sys.exit(
+            "Invalid username, password or secret key - please check and retry. If problem persists contact helpdesk on helpdesk@ega-archive.org")
 
     return oauth_token
 
+
 def api_list_authorized_datasets(token):
     """List datasets to which the credentialed user has authorized access"""
 
-    headers = {'Accept':'application/json', 'Authorization': 'Bearer {}'.format(token)}
-    headers.update( get_standart_headers() )
+    headers = {'Accept': 'application/json', 'Authorization': 'Bearer {}'.format(token)}
+    headers.update(get_standart_headers())
 
-    url = URL_API+"/metadata/datasets"
+    url = URL_API + "/metadata/datasets"
     r = requests.get(url, headers=headers)
     r.raise_for_status()
 
     reply = r.json()
 
-    print_debug_info(url,reply)
+    print_debug_info(url, reply)
 
     if reply is None:
-        sys.exit("You do not currently have access to any datasets at EGA according to our databases. If you believe you should have access please contact helpdesk on helpdesk@ega-archive.org")
+        sys.exit(
+            "You do not currently have access to any datasets at EGA according to our databases. If you believe you should have access please contact helpdesk on helpdesk@ega-archive.org")
 
     return reply
 
+
 def pretty_print_authorized_datasets(reply):
     print("Dataset ID")
     print("-----------------")
     for datasetid in reply:
         print(datasetid)
 
+
 def api_list_files_in_dataset(token, dataset):
-    if( dataset in LEGACY_DATASETS):
+    if (dataset in LEGACY_DATASETS):
         sys.exit("This is a legacy dataset {}. Please contact the EGA helpdesk for more information.".format(dataset))
 
-    headers = {'Accept':'application/json', 'Authorization': 'Bearer {}'.format(token)}
-    headers.update( get_standart_headers() )
-    url = URL_API+"/metadata/datasets/{}/files".format(dataset)
+    headers = {'Accept': 'application/json', 'Authorization': 'Bearer {}'.format(token)}
+    headers.update(get_standart_headers())
+    url = URL_API + "/metadata/datasets/{}/files".format(dataset)
 
-    if( not dataset in api_list_authorized_datasets(token) ):
+    if (not dataset in api_list_authorized_datasets(token)):
         sys.exit("Dataset '{}' is not in the list of your authorized datasets.".format(dataset))
 
-    r = requests.get(url, headers = headers)
+    r = requests.get(url, headers=headers)
     r.raise_for_status()
     logging.debug(r)
 
     reply = r.json()
-    print_debug_info(url,reply)
+    print_debug_info(url, reply)
 
     if reply is None:
```

Comment:
Question: Does this "None" that the Dataset does not exist? If it's like I suggest to improve the error text and make it more accurate.

Reply:
I am not much aware about python syntax but I found this "The sole value of the type NoneType. None is frequently used to represent the absence of a value, as when default arguments are not passed to a function. Assignments to None are illegal and raise a SyntaxError."